EasyPost.Tests/UserTest.cs

```diff
@@ -1,102 +1,124 @@
+                    User retrievedUser = await Client.User.Retrieve(id);
                     await retrievedUser.Delete();
-                    _userId = null;
+                    return true;
                 }
                 catch
                 {
-                    // in case we try to delete something that's already been deleted
+                    // trying to delete something that doesn't exist, pass
+                    return false;
                 }
-            }
-        }
+            };
 
-        private static async Task<User> RetrieveMe()
+        #region CRUD Operations
+
+        [Fact]
+        [CrudOperations.Create]
+        public async Task TestCreate()
         {
-            return await User.RetrieveMe();
+            UseVCR("create");
+
+            User user = await CreateUser();
+
+            Assert.IsType<User>(user);
+            Assert.StartsWith("user_", user.id);
+            Assert.Equal("Test User", user.name);
         }
 
-        private static async Task<User> CreateUser()
+        [Fact]
+        [CrudOperations.Create]
+        public async Task TestUpdateBrand()
         {
-            User user = await User.Create(new Dictionary<string, object>
+            UseVCR("update_brand");
+
+            User user = await CreateUser();
+
+            string color = "#123456";
+            Brand brand = await user.UpdateBrand(new Dictionary<string, object>
             {
                 {
-                    "name", "Test User"
+                    "color", color
                 }
             });
-            _userId = user.id; // trigger deletion after test
-            return user;
+
+            Assert.IsType<Brand>(brand);
+            Assert.StartsWith("brd_", brand.id);
+            Assert.Equal(color, brand.color);
         }
 
-        // This endpoint returns the child user keys in plain text, do not run this test.
-        [TestMethod]
-        public async Task TestCreate()
+        [Fact]
+        [CrudOperations.Read]
+        public async Task TestAllApiKeys()
         {
-            _vcr.SetUpTest("create");
+            UseVCR("all_api_keys");
 
-            User user = await CreateUser();
+            List<ApiKey> apiKeys = await Client.ApiKey.All();
 
-            Assert.IsInstanceOfType(user, typeof(User));
-            Assert.IsTrue(user.id.StartsWith("user_"));
-            Assert.AreEqual("Test User", user.name);
+            // API keys will be censored, so we'll just check for the existence of the list
+            Assert.NotNull(apiKeys);
         }
 
-        [TestMethod]
+        [Fact]
+        [CrudOperations.Read]
+        public async Task TestApiKeys()
+        {
+            UseVCR("api_keys");
+
+            User user = await RetrieveMe();
+
+            // API keys will be censored, so we'll just check for the existence of the `children` element
+            List<User> children = user.children;
+            Assert.NotNull(children);
+        }
```

Comment:
Issue: This test is bogus and should be removed...

Reply:
Resolved.